devops/templates/dokku-nginx.conf

```diff
@@ -1,25 +1,55 @@
 server {
     listen       443 ssl;
-    server_name  {{ dokku_domains | join(" ") }};
+    server_name  {{ dokku_wildcard_domains | join(" ") }};
 
     access_log /var/log/nginx/dokku.log;
     client_max_body_size 100M;
 
-    ssl_certificate /etc/letsencrypt/live/dokku/fullchain.pem;
-    ssl_certificate_key /etc/letsencrypt/live/dokku/privkey.pem;
+    ssl_certificate /etc/letsencrypt/live/dokku-wildcard/fullchain.pem;
+    ssl_certificate_key /etc/letsencrypt/live/dokku-wildcard/privkey.pem;
 
     location / {
         proxy_pass http://{{ dokku_proxy_host }}:{{ dokku_proxy_port }};
         proxy_set_header Host $host;
         proxy_set_header X-Real-IP $remote_addr;
         proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
         proxy_set_header X-Forwarded-Proto "https";
+
+        proxy_http_version 1.1;
+        proxy_set_header Upgrade $http_upgrade;
+        proxy_set_header Connection $http_connection;
+        proxy_read_timeout 60s;
+        proxy_buffering off;
```

Comment:
For performance and caching reasons, it is probably better to keep `proxy_buffering` enabled when WebSockets are not used.